app/views/layouts/body/_navbar_section.html.erb

```diff
@@ -1,41 +1,67 @@
 <% if @section.present? %>
-<% if @section == "contemporarypoets" %>
-<div class="navbar navbar-inverse">
-  <div class="container navbar-container">  
-    <ul class="nav navbar-nav">
-      <li class="<%= active?(home_path) %>">
-        <%= link_to "Contemporary Poets Home", prefix_path("home_path") %>
-      </li>
-       <li class="<%= active?(/^#{config.relative_url_root}\/browse/,
-        request.path) %>">
-        <%= link_to "Featured", featured_path %>
-      </li>
-      <li class="<%= active?(/^#{config.relative_url_root}\/browse/,
-        request.path) %>">
-        <%= link_to t("browse.title", default: "Browse"), prefix_path("browse_path") %>
-      </li>
-      <li class="<%= active?(search_path) %>">
-        <%= link_to t("search.title", default: "Search"), prefix_path("search_path"), rel: "search" %>
-      </li>
-    </ul>
-  </div>
-</div><!-- /navbar -->
-<% end %>
-<% if @section == "about" %>
-<div class="navbar navbar-inverse">
-  <div class="container navbar-container">
-    <ul class="nav navbar-nav">
-      <li>
-        <%= link_to "Credits", about_homeabout_path %>
-      </li>
-      <li>
-        <%= link_to "African Poetry Book Fund", about_africanpoetrybookfund_path %>
-      </li>
-      <li>
-        <%= link_to "Technical Details", about_technicaldetails_path %>
-      </li>
-    </ul>
-  </div>
-</div>
-<% end %>
+  <% if @section == "contemporarypoets" %>
+    <div class="navbar navbar-inverse">
+      <div class="container navbar-container">  
+        <ul class="nav navbar-nav">
+          <li class="<%= active?(home_path) %>">
+            <%= link_to "Contemporary Poets Home", prefix_path("home_path") %>
+          </li>
+           <li class="<%= active?(/^#{config.relative_url_root}\/browse/,
+            request.path) %>">
+            <%= link_to "Featured", featured_path %>
+          </li>
+          <li class="<%= active?(/^#{config.relative_url_root}\/browse/,
+            request.path) %>">
+            <%= link_to t("browse.title", default: "Browse"), prefix_path("browse_path") %>
+          </li>
+          <li class="<%= active?(search_path) %>">
+            <%= link_to t("search.title", default: "Search"), prefix_path("search_path"), rel: "search" %>
+          </li>
+        </ul>
+      </div>
+    </div><!-- /navbar -->
+  <% end %>
+  <% if @section == "about" %>
```

Comment:
I believe that these could all be `elsif` type of sections but probably it's not that critical, it just means that Rails won't have to check every "if" when it gets an earlier match

Reply:
This is fine for now, I think this could be completely redone in a better way anyway